### PR TITLE
[Issue: 1291] Update cd-* scripts to deploy staging

### LINKED ISSUE #1291: [Task]: Update `cd-*` scripts to deploy staging
### Summary

The existing `cd-api`, `cd-frontend`, `cd-analytics` scripts should also deploy staging.

~Discuss with the team around whether or not they should _**also**_ continue to deploy dev.~ done!

### Acceptance criteria

- [ ] the `cd-*` scripts have been updated to deploy staging

### LINKED ISSUE #1549: [Task]: Delivery Dashboard - Deploy application with CI/CD
### Summary

Deploy the analytics application with CI/CD, via a `cd-analytics` file

### Acceptance criteria

- [ ] `cd-analytics` file works in all environments

### PR DESCRIPTION
**Pull request by aplybeah**
## Summary
Fixes #1291
Fixes #1549 

## Changes proposed
* add `staging` input to `cd-api` and `cd-frontend` files

## Additional information
N/A

### CODE REVIEW

**Review by chouinar — DISMISSED** [review]

File: .github/workflows/cd-frontend.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ on:
         type: choice
         options:
           - dev
+          - staging
```
Comment (coilysiren) [inline .github/workflows/cd-frontend.yml:23]: Unfortunately adding `staging` here won't change much. It just makes staging available as an option, but doesn't actually start deploying staging. The line you want to change is down below...

File: .github/workflows/cd-frontend.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@ on:
...
           - prod
```
Comment (coilysiren) [inline .github/workflows/cd-frontend.yml:25]: ...the line you want to change is this one down below

```
environment: ${{ inputs.environment || (github.event_name == 'release' && 'prod') || 'dev' }}
```

change it to (psuedo-code)

```
environment: ${{ inputs.environment || (github.event_name == 'release' && 'prod') || ['dev', 'staging'] }}
```

Comment (coilysiren) [reply .github/workflows/cd-frontend.yml:25]: You'll need to do this for the analytics package as well, of course.

**Review by coilysiren — CHANGES_REQUESTED** [review]

File: .github/workflows/cd-frontend.yml [hunk 3]
```diff
@@ -20,6 +20,7 @@ on:
         type: choice
```
Comment (coilysiren) [inline .github/workflows/cd-frontend.yml:20]: You'll need to change `run-name:` above as well

**Review by coilysiren — APPROVED** [review]
This looks right 👍🏼 I would just recommend testing it before merging

**Pull request merged**